src/confighttp.cpp

```diff
@@ -298,6 +304,11 @@ void saveApp(resp_https_t response, req_https_t request) {
     pt::read_json(ss, inputTree);
     pt::read_json(config::stream.file_apps, fileTree);
 
+    // Moonlight checks the id of an item to determine if an item was changed
+    // Add a property named "id" to the inputTree with a value of the current timestamp in milliseconds
+    long long id = duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count();
```

Comment:
Some Moonlight clients (Android at least) expect the ID to be within the range of a 32-bit signed int. I'd truncate this to avoid exceeding that range.